### DISCUSSION TITLE
Latest update removes key display from skins

### DISCUSSION THREAD

**Discussion by Non-Binary-Programmer** [ppy/osu#24254] [Q&A]
### Type

Game behaviour

### Bug description

Upon updating lazer to the most recent version, my skin, which I had edited using the built in skin editor to include the "triangles" key display, had the key display removed from it. Other edits I made to the skin, such as adding the "triangles" pp counter, remained.

### Version

updating to 2023.716.0-lazer from previous release

### Logs

[updater.log](https://github.com/ppy/osu/files/12064511/updater.log)
[database.log](https://github.com/ppy/osu/files/12064512/database.log)
[input.log](https://github.com/ppy/osu/files/12064513/input.log)
[network.log](https://github.com/ppy/osu/files/12064514/network.log)
[performance.log](https://github.com/ppy/osu/files/12064515/performance.log)
[runtime.log](https://github.com/ppy/osu/files/12064516/runtime.log)

**Comment by bdach** [comment]
Please attach skin.

**Comment by longnguyen2004** [comment]
If I recall correctly, the key display wasn't a skin element before the latest update, so it wasn't present in the skin settings to begin with. Maybe the dev can automatically add it in if it's not already present in the skin, but I'm fine with the current behavior.

**Comment by peppy** [comment]
I think you will need to add it back manually.
